lib/manageiq/loggers/deprecator.rb

```diff
@@ -0,0 +1,7 @@
+module ManageIQ
+  module Loggers
+    def self.deprecator
+      @deprecator ||= ActiveSupport::Deprecation.new(VERSION, "ManageIQ::Loggers")
```

Comment:
Does this code need to require "active_support" or "active_support/deprecation" now?